org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/alive/ConsumerBean.java

```diff
@@ -14,17 +14,18 @@
 import org.eclipse.scanning.api.INameable;
 
 /**
- * 
+ *
  * Bean send on the Constants.ALIVE_TOPIC to notify every now and then
- * 
- * 
+ *
+ *
  * @author Matthew Gerring
  * @deprecated replaced by HeartbeatBean. This is here in case new clients look at old consumers.
  */
+@Deprecated
 public class ConsumerBean implements INameable {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)